test/e2e_federation/federated-service.go

```diff
@@ -92,15 +92,15 @@ var _ = framework.KubeDescribe("Federated Services [Feature:Federation]", func()
 			clusters, primaryClusterName = getRegisteredClusters(UserAgentName, f)
 		})
 
-		Describe("service creation", func() {
+		Describe("Federated Service", func() {
```

Comment:
@shashidharatd Wow! Excellent catch! Is this PR independent of your service controller refactor? Can we get this in ASAP?